scaling_performance/using_cluster_loader.adoc

```diff
@@ -181,9 +193,33 @@ defined in `stepsize`.
 not successful.
 
 |`delay`
-|An integer. How many milliseconds (ms) to wait between creation requests
+|An integer. How many milliseconds (ms) to wait between creation requests.
+|===
+
+.Fields under `sync`
+|===
+|Field |Description
+
+|`server`
+|A sub-object with `enabled` and `port` fields. Boolean `enabled` defines
+whether to start a HTTP server for pod synchronization.  Integer `port`
+defines HTTP server port to listen on (9090 by default).
+
+|`running`
+|A boolean. Wait for pods with labels matching `selectors` to go into Running state?
```

Comment:
Is there a reason why these are punctuated by ?

Reply:
Like `selectors`?  Yes, because I'm trying to refer to the `selectors` below.  Could be phrased better I guess.